extras/CatchAddTests.cmake

```diff
@@ -119,43 +123,56 @@ function(catch_discover_tests_impl)
 
   # Parse output
   foreach(line ${output})
-    set(test ${line})
-    # Escape characters in test case names that would be parsed by Catch2
-    set(test_name ${test})
-    foreach(char , [ ])
-      string(REPLACE ${char} "\\${char}" test_name ${test_name})
-    endforeach(char)
-    # ...add output dir
-    if(output_dir)
-      string(REGEX REPLACE "[^A-Za-z0-9_]" "_" test_name_clean ${test_name})
-      set(output_dir_arg "--out ${output_dir}/${output_prefix}${test_name_clean}${output_suffix}")
-    endif()
-
-    # ...and add to script
-    add_command(add_test
-      "${prefix}${test}${suffix}"
-      ${_TEST_EXECUTOR}
-      "${_TEST_EXECUTABLE}"
-      "${test_name}"
-      ${extra_args}
-      "${reporter_arg}"
-      "${output_dir_arg}"
-    )
-    add_command(set_tests_properties
-      "${prefix}${test}${suffix}"
-      PROPERTIES
-      WORKING_DIRECTORY "${_TEST_WORKING_DIR}"
-      ${properties}
-    )
+    if(line MATCHES "^  ([^ ].*)$")
```

Comment:
This is gonna need more comments to explain why this looks for exactly two spaces and so on.

Reply:
for sure, but is something like
```
# Catch2's binaries, when running with '--list-tests --verbosity normal', will print out the name of the tests and their tags. The tests' names are indented with 2 spaces and the tags are indented with 4 spaces
```
ok? Is this also a guarantee from Catch2?